.github/workflows/build-images.yaml

```diff
@@ -73,11 +73,116 @@ jobs:
       - name: Tag and push image
         run: |
           for input_tag in ${{ inputs.tags }}; do
-            docker tag trino:472-amd64 onehouse/onehouse-trino:$input_tag
-            docker push onehouse/onehouse-trino:$input_tag
+            docker tag trino:472-amd64 onehouse/onehouse-trino:amd-$input_tag
+            docker push onehouse/onehouse-trino:amd-$input_tag
```

Comment:
nit: better to use suffix in the naming to be consistent: `${input_tag}-amd`

Reply:
Yes, this is for consistency in naming as Trino generates image as `472-amd64`, even if these images are hidden.